ui/e2e/src/tests/persesApp.spec.ts

```diff
@@ -26,4 +26,38 @@ test.describe('App', () => {
     await homePage.clickDashboardItem('Demo');
     await navigationPromise;
   });
+
+  test('can create a new dashboard', async ({ page }) => {
+    const homePage = new AppHomePage(page);
+    await homePage.goto();
+
+    // Go to testing project
+    await homePage.showDashboardList('testing');
+    const projectLink = await page.getByRole('link', { name: 'testing' });
+    const navigationPromise = page.waitForNavigation();
+    await projectLink.click();
+    await navigationPromise;
+
+    // Create a new dashboard
+    const projectPage = new AppProjectPage(page);
+    await projectPage.createDashboard('my new dashboard');
+
+    // Should see empty state
+    await expect(page.getByRole('main')).toContainText("Let's get started");
```

Comment:
do you think it makes sense to add a happo screenshot of what the empty state looks like?

Reply:
We get a bunch of them from the stories for the empty state, so I was thinking that was enough, but it's probably a good idea to get one from the app too, since it's customized. Will add one.